karibu-dsl/src/main/kotlin/com/github/mvysny/karibudsl/v10/KLitRendererAttributesDsl.kt

```diff
@@ -0,0 +1,97 @@
+package com.github.mvysny.karibudsl.v10
+
+import com.vaadin.flow.component.icon.Icon
+import com.vaadin.flow.component.shared.ThemeVariant
+import kotlinx.css.CssBuilder
+import kotlinx.css.hyphenize
+
+public data class KLitRendererAttribute(
+    val name: String,
+    val value: String
+) {
+    override fun toString(): String = "$name=\"$value\""
+}
+
+/**
+ * style="align-items: center;"
+ * style="line-height: var(--lumo-line-height-m);"
+ * style="font-size: var(--lumo-font-size-s); color: var(--lumo-secondary-text-color);"
+ */
+private fun CssBuilder.buildInlineStyleText() =
+    declarations.asSequence().joinToString(separator = "; ") { (key, value) ->
+        "${key.hyphenize()}: $value"
+    }
+
+public fun <TSource> KLitRendererTagsBuilderA<TSource>.style(value: String) =
+    KLitRendererAttribute("style", value)
+
+public fun <TSource> KLitRendererTagsBuilderA<TSource>.style(block: CssBuilder.() -> Unit) =
+    KLitRendererAttribute("style", CssBuilder().apply(block).buildInlineStyleText())
+
+public fun <TSource> KLitRendererTagsBuilderA<TSource>.theme(vararg names: KLitRendererTheme) =
+    KLitRendererAttribute("theme", names.joinToString(separator = " ") { it.name })
+
+public fun <TSource> KLitRendererTagsBuilderA<TSource>.themeVariant(vararg names: ThemeVariant) =
+    KLitRendererAttribute("theme", names.joinToString(separator = " ") { it.variantName })
+
+public fun <TSource> KLitRendererTagsBuilderA<TSource>.cssClass(vararg names: String) =
+    KLitRendererAttribute("class", names.joinToString(separator = " "))
+
+public fun <TSource> KLitRendererTagsBuilderA<TSource>.id(value: String) =
+    KLitRendererAttribute("id", value)
+
+/**
+ * --- Renderers https://vaadin.com/docs/latest/components/grid/renderers
```

Comment:
Could you add an example on how to use this function and what that usage will produce?